06.event_listeners/spec/eventListeners.js

```diff
@@ -124,17 +134,22 @@ describe('EventListeners', function() {
 
   it('should delegate an event to elements with a given css class name', function() {
     eventListener.delegate(selectedElement, 'title', 'click', methods.showLove);
+    var titleEl = document.querySelector('.title');
 
-    $('.title').trigger('click');
-
+    var eClick = new Event('click');
+    titleEl.dispatchEvent(eClick);
+    
     expect(methods.showLove.calls.count()).toEqual(1);
   });
 
   it('should not delegate an event to elements without a given css class name', function() {
     eventListener.delegate(selectedElement, 'title', 'click', methods.showLove);
+    var titleEl = document.querySelector('.title');
+    var subtitleEl = document.querySelector('.subtitle');
 
-    $('.subtitle').trigger('click');
-    $('.title').trigger('click');
+    var eClick = new Event('click');
+    titleEl.dispatchEvent(eClick);
+    subtitleEl.dispatchEvent(eClick);
```

Comment:
It should be fine in this example, but you are correct. I'll create a new event.